bindings/wasm/examples/editor.html

```diff
@@ -0,0 +1,192 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <title>Manifold - <code></code>&lt;model-viewer&gt;</code> Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <style>
+    body {
+      margin: 0;
+      background-color: #f7f7f7;
+      font-family: 'Rubik', sans-serif;
+      font-size: 16px;
+      line-height: 24px;
+      color: rgba(0, 0, 0, .87);
+      font-weight: 400;
+      -webkit-font-smoothing: antialiased;
+    }
+
+    p {
+      max-width: 700px;
+      margin: 1em;
+      text-align: left;
+    }
+
+    .container {
+      display: flex;
+      height: 80vh;
+    }
+
+    .col {
+      flex: 1;
+      height: 100%;
+    }
+  </style>
+</head>
+
+<body>
+  <p>This example demonstrates feeding Manifold's output into <a
+      href="https://modelviewer.dev/">&lt;model-viewer&gt;</a>
+    via the three.js GLTFExporter. The most basic three.js example is <a href="index.html">here</a>.
+  </p>
+  <button type="button" id="compile">Compile</button>
+  <div class="container">
+    <model-viewer class="col" camera-controls shadow-intensity="1" alt="Output of mesh Boolean operation"></model-viewer>
+    <div id="editor" class="col">
+    </div>
+  </div>
+</body>
+
+<script src="https://unpkg.com/three@0.144.0/build/three.js"></script>
+<script src="https://unpkg.com/three@0.144.0/examples/js/utils/BufferGeometryUtils.js"></script>
+<script src="https://unpkg.com/three@0.144.0/examples/js/exporters/GLTFExporter.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.34.0/min/vs/loader.min.js" integrity="sha512-6bIYsGqvLpAiEBXPdRQeFf5cueeBECtAKJjIHer3BhBZNTV3WLcLA8Tm3pDfxUwTMIS+kAZwTUvJ1IrMdX8C5w==" crossorigin="anonymous" referrerpolicy="no-referrer"></script>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.34.0/min/vs/editor/editor.main.min.css" integrity="sha512-iQEIc0rsSDujsfjtD+lfyJ1W23Bh/lbgriubKDAym6VlEIDRj9rrbSIyJRyshOrl8s0yRcQ0+gyrZfSLyjJGWQ==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+<script>
+  var editor = undefined;
+  var Module = {
```

Comment:
`var` is old-school; generally we try to just use `const` and `let` now.

Reply:
Yeah I kind of messed up the styles when copying the example from monaco repo.

Reply:
For the `Module` one, I'm not sure what is the appropriate way to do it: the wasm js file is loaded after this script, which will redefine `Module`, so we cannot change this to `let`. Perhaps we should just move this script section after loading wasm js files?

Reply:
That's fine, or if this is a legit use of `var`, we can just leave it. I was mostly curious.